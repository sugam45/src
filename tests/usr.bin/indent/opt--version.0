/* $NetBSD: opt--version.0,v 1.2 2021/03/06 19:30:44 rillig Exp $ */
/* $FreeBSD$ */

When the option '--version' is given, any other options are ignored.
Therefore the source file, if given, can contain arbitrary text that
even might generate syntax errors when given to a C compiler.
