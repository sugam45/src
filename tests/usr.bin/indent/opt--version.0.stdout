FreeBSD indent 2.0
